requirements/testing.txt

```diff
@@ -8,23 +8,39 @@
 -r base.txt
 -r integration.txt
 -e file:.                 # via -r requirements/base.in
+appnope==0.1.0            # via ipython
 astroid==2.4.2            # via pylint
+backcall==0.2.0           # via ipython
 coverage==5.2.1           # via pytest-cov
+docker==4.3.0             # via -r requirements/testing.in
 flask-testing==0.8.0      # via -r requirements/testing.in
 iniconfig==1.0.1          # via pytest
+ipdb==0.13.3              # via -r requirements/testing.in
+ipython-genutils==0.2.0   # via traitlets
+ipython==7.16.1           # via -r requirements/testing.in, ipdb
```

Comment:
This is surprising, do we need this dependency?

Reply:
this is for ipdb, it CI had issues locationg ipython > 7.16.1, I'll try it again.
As for ipdb - I like using it inside tox, that's why added to the test dependencies. It provides really nice debugging experience.

Reply:
*update* github actions still are unable to install ipython 7.17.0

Reply:
Nice, I'll try it out 👍